sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/APISpec.groovy

```diff
@@ -90,7 +90,7 @@ class APISpec extends Specification {
 
     static final String garbageLeaseID = UUID.randomUUID().toString()
 
-    public static final String defaultEndpointTemplate = "https://%s.dfs.core.windows.net/"
+    public static final String defaultEndpointTemplate = "http://%s.dfs.core.windows.net/"
```

Comment:
You'll want to change this back if this was being used to test out running with a proxy. Reach out if you want a guide which steps through setting up a Fiddler proxy with HTTPS enabled.

Reply:
Done